frontend/apps/core-app/src/components/RecordTable/index.ts

```diff
@@ -0,0 +1,4 @@
+import { RecordTableContainer } from './RecordTableContainer';
+
+export default RecordTableContainer;
```

Comment:
Calling this component `RecordTable` still indicates some level of generalisation, but the use of the context prevents multiple instances being used. I don't think context is needed here, but if you want to keep it I would make one of these changes:

* Rename it to `RecipientListTable` so that it has a single use
or
* Change the context to be a function that generates a context. So whenever you create a new table a new context is created